tests/testthat/test-derive_vars_dt.R

```diff
@@ -569,4 +635,4 @@ test_that("derive_vars_dt Test 24: Supplying both min/max dates for highest_impu
   expect_dfs_equal(actual, expected, keys = c("ASTDT", "ASTDTF"))
 })
 
-rm(input, input_warnings)
+rm(input, input_warnings, inputdt, inputdtc, date)
```

Comment:
Could you add issue in our Discussion to look at a way to adopt this in all our testing scripts?  maybe it is already there!